sound/soc/sof/loader.c

```diff
@@ -12,7 +12,7 @@
 
 #include <linux/firmware.h>
 #include <sound/sof.h>
-#include <uapi/sound/sof/ext_manifest.h>
+#include <sound/sof/ext_manifest.h>
```

Comment:
interesting, so without this new header just doing `#include <sound/sof/ext_manifest.h>` couldn't find the header? I wasn't actually proposing a new header, I thought include/uapi was among the kernel build system search paths, is it not?

Reply:
@lyakh This seems a stale comment now, please reopen if stll valid.